Options.py

```diff
@@ -1084,7 +1084,11 @@ def from_any(cls, data: PlandoConFromAnyType) -> Self:
                 percentage = connection.get("percentage", 100)
                 if random.random() < float(percentage / 100):
                     entrance = connection.get("entrance", None)
+                    if is_iterable_except_str(entrance):
+                        entrance = random.choice(entrance)
                     exit = connection.get("exit", None)
+                    if is_iterable_except_str(exit):
+                        exit = random.choice(exit)
```

Comment:
This would probably work for most of what we'll get from yamls...
But `__iter__` is not what `random.choice` needs.
`random.choice` needs `__len__` and `__getitem__` (`Sequence`)
I'm not sure whether we need a `is_sequence_except_str`...

```suggestion
                    entrance = connection.get("entrance", None)
                    if entrance and not isinstance(entrance, str):
                        entrance = random.choice(entrance)
                    exit = connection.get("exit", None)
                    if exit and not isinstance(exit, str):
                        exit = random.choice(exit)
```
We should think about the code path if we get something that is neither a `str` nor a `Sequence`
I think that could be a problem with or without this change.

Reply:
I think without manually invoking `from_any`, the only possible Iterable that isn't a Sequence would be a set, in which case we could wrap in `sorted`? You'd have to go out of your way to say this is a set and not a list in the yaml though.

Reply:
`entrance = random.choice(sorted(entrance))`
That works with the iterable check, since it'll take any `Iterable` and turn it into `Sequence`